src/components/home/partners/LogoComponent.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+
+const styles = theme => ({
+  root: {
+    marginBottom: 15
+  },
+  logo: {
+    position: 'relative',
+    width: '100%'
+  }
+})
+
+class LogoComponent extends Component {
+  render() {
+    const { Logo } = this.props
+    const { classes } = this.props;
+
+    return (
+      <Grid
+        item
+        xs={5}
+        sm={5}
+        className={classes.root}
+      >
+        <div className={classes.logo}>
+          {Logo}
```

Comment:
For this class, consider passing in `filename` as a prop, instead of passing an entire component. So it could be like `<Logo filename={} />`. Then you just render the `<Image>` component inside `Logo` class. Then the styles also can be shifted to here from `PartnersComponent`

Reply:
resolved.